meta-balena-common/recipes-support/hostapp-update-hooks/files/99-resin-grub

```diff
@@ -75,7 +76,7 @@ else
         fi
 
         EFI_BINARY_SECUREBOOT="${EFI_BINARY}.secureboot"
-        if [ "${type}" = "crypt" ]; then
+        if [ "${secureboot}" = "true" ]; then
```

Comment:
nitpick: it's not really checking for secure boot though, is it? It's just checking whether the partitions are encrypted. I went for `enc_part` or similar in my PR.